### ISSUE TITLE
Dancing Text in Nested List

### ISSUE THREAD

**Issue by jar617** [Zettlr/Zettlr#4690]
### Description

I have an issue that emerged only after updating to Zettlr 3.0: dancing text with nested lists. 

It's a little hard to describe in words, so I have uploaded a short screen capture. The issue involves nested lists. Start a list as normal and then add a nested list item - when typing the nested list item, the text for that item (but not the other items in the list) starts jumping around erratically. In addition, this occurs even when not modifying the list itself (e.g., when writing something after a line break below the list). I had Zettlr (pre 3.0.1) installed on this very same computer without issue, so this is something related to the new release. 

Is there any potential solution? 

https://github.com/Zettlr/Zettlr/assets/60667084/a4190bf5-2216-4a29-a805-1cc58bbdf8c7

### Reproducing

Create a nested list. 

### Zettlr Version

Stable (most recent version)

### Specify version

3.0.1

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
Duplicate of #4602
